src/app/menu/menu.component.html

```diff
@@ -31,40 +31,26 @@
         </div>
         <div class="row">
             <div class="menu-button">
-                <img class="hex" src="../../assets/img/meds.png" (click)="navigateToPage('/meds-n-labels')">
-                <button class="but med-btn" (click)="navigateToPage('/meds-n-labels')">
+                <img class="hex" [src]="calculateButtonBackground(3, 'meds')" (click)="navigateToPage('/meds-n-labels')" >
+                <button [ngClass]="calculateButtonClasses(3, 'med-btn')" (click)="navigateToPage('/meds-n-labels')" [disabled]="!checkStageStatus(3)">
                     {{language?.medsNLabels.heading}}
                 </button>
-                <div class="outer-circle med-outer-circle">
-                    <div class="inner-circle med-inner-circle">
+                <div [ngClass]="calculateOuterClasses(3, 'med-outer-circle')">
+                    <div [ngClass]="calculateInnerClasses(3, 'med-inner-circle')">
                         <div *ngFor="let level of createRange(3); let idx=index" class="arc level{{idx+1}}" [ngClass]="stage>3 || (stage===3 && activity>=idx+1) ? 'completed' : ''"></div>
```

Comment:
please create a method for this 
(stage===3 && activity>=idx+1) ? 'completed' : ''"
make it return a boolean

Reply:
@shreyans29 this was implemented like this, last year. I didn't add it. Will add the method for it.